ApplePicker Error in upstream software stack

In #181 , I cleaned up existing warnings and also checked upstream warnings (those with newer versions of dependencies than we currently employ).  Upon letting python take the latest packages it saw fit, I was greeted with the following error:

`ApplePickerTestCase.testPickCenters failed with AssertionError: (1321.0, 361.0) not an expected center.`

I think we'd need to dig into that.

Generally, people seem satisfied with the function of ApplePicker.  There may be one or two small ApplePicker issues around, and so it might make sense to have an ApplePicker development iteration later after other priorities are addressed. We can stand up a project board for those issues at that time.

--- a/src/aspire/apple/picking.py
+++ b/src/aspire/apple/picking.py
@@ -88,7 +88,8 @@ class Picker:
         im = im[:side_length, :side_length]
 
         size = tuple((np.array(im.shape) / config.apple.mrc_shrink_factor).astype(int))
-        im = np.array(Image.fromarray(im).resize(size, Image.BICUBIC))
+
+        im = np.array(Image.fromarray(im).resize(size, Image.BICUBIC), dtype=np.float64)
 
         im = signal.correlate(
             im,
@@ -99,7 +100,7 @@ class Picker:
             'same'
         )
 
-        return im.astype('double')
+        return im
 
     def query_score(self, show_progress=True):
         """Calculates score for each query image.

